content/en/_index.md

```diff
@@ -40,30 +41,57 @@ These principles apply in descending order of priority:
 6. **Maintainability**: The code is written such that it can be easily
    maintained
 
-Code snippets should use preferred code patterns that are idiomatic to the
+{{% alert title="Considering idiomatic code vs. consistent samples across languages" %}}
+Code samples should use preferred code patterns that are idiomatic to the
 particular language (e.g., promises, futures, static API calls from singletons,
-builders, etc). This may result in code snippets that are radically different
+builders). This may result in samples that are radically different
 from language-to-language. Err on the side of what is idiomatic to the language,
 though cross-language consistency does not hurt. Generally, do things
 "the way the community does it."
+{{% /alert %}}
+
+### Sample Categories
+
+There are two categories of samples covered in this guide:
+
+* **✂︎ Snippets:** A snippet is a standalone slice of code that demonstrates
+  a specific Cloud operation.
+* **🧩 Patterns:** A pattern demonstrates a use case through multiple slices of
+  code, such as a sample application. Individual slices should follow
+  [snippet structure guidelines](#structure) where it improves code clarity.
 
-## Structure {#structure}
+## Metadata {#metadata}
 
 ### Region tags {#region-tags}
 
-Each code snippet should have a region tag to define which parts of the snippet
-are displayed from the documentation. Each region tag should:
+**Applies to:** ✂︎ Snippets, 🧩 Patterns
+
+Region tags define a slice of code as a snippet and which parts of the code are
+displayed from the documentation.
+
+#### Each region identifier
 
 * Be globally unique
 * Be consistent across the same snippets in different languages
 * Begin with the product's region tag prefix
 * Use snake case (`snake_case`)
 
-Region tags should show as much of the sample as possible, so that a user can
+#### When to use region tags
+
+* **✂︎ Snippets:** Region tags should contain as much of the sample as possible, so that a user can
 easily copy and paste the sample into their own environment to run it.
+* **🧩 Patterns:** Region tags should contain those slices of code that are shown in documentation.
+
+#### Changing region tags
+
+Region tags and the scope of code they contain are a documentation dependency.
+
+## Snippet Structure {#structure}
 
 ### Imports {#imports}
 
+**Applies to:** ✂︎ Snippets
```

Comment:
This may be applicable to patterns as well. I understand that not always, but maybe a hint like:

**Applies to on best effort: ... Patterns

Reply:
Added (best effort) to patterns note and left at that for now. We may want more nuanced explanation later.